Docker_run_cuda

```diff
@@ -0,0 +1,30 @@
+ARG BUILD_IMAGE_NAME
```

Comment:
how about having this file in the docker folder?

Reply:
Other Docker files are in the main folder so I wanted to keep it that way to be consistent.